hack/lib/util.sh

```diff
@@ -531,6 +531,16 @@ EOF
 EOF
 }
 
+# list_staging_repos outputs a sorted list of repos in staging/src/k8s.io
+# each entry will just be the $repo portion of staging/src/k8s.io/$repo/...
+function kube::util::list_staging_repos() {
```

Comment:
originally I didn't add this to util, but all 3 fixed scripts have a need for this and doing it correctly / portably is tricky